components/base/kinematicbase/execution.go

```diff
@@ -250,6 +251,7 @@ func (ptgk *ptgBaseKinematics) trajectoryToArcSteps(
 				inputs[0],
 				inputs[1],
 				{curDist},
+				inputs[3],
```

Comment:
[minor]: I think we should display uniformity here with how we access/refer to these values.
Why not:
```
[]referenceframe.Input{
    inputs[trajectoryIndexWithinPTG],
    inputs[startDistanceAlongTrajectoryIndex],
    {curDist},
    inputs[endDistanceAlongTrajectoryIndex],
}
```